src/validation-schema/index.ts

```diff
@@ -0,0 +1,54 @@
+import { z } from 'zod';
+// import { LogLevel } from '../config';
+import { TaskPriority } from '../repositories/task/create';
+
+const portRegex = /^(?:[1-9]\d{0,3}|[1-5]\d{4}|6[0-4]\d{3}|65[0-4]\d{2}|655[0-2]\d|6553[0-5])$/;
+const validatePort = (port: string) => {
+  if (!portRegex.test(port)) {
+    throw new Error('Invalid PORT number');
+  }
+
+  return Number(port);
+};
+
+export const DBConfigValidationSchema = z.object({
+  url: z.string()
+});
+
+export const ConfigValidationSchema = z.object({
+  appEnv: z.string(),
+  dbConfig: z.string().transform((val) => DBConfigValidationSchema.safeParse(JSON.parse(val))),
```

Comment:
What happens when I set `dbConfig` to '['? It may be hard to understand the root cause